### ISSUE TITLE
Make the buttons in the web player more accessible to screen readers

### ISSUE THREAD

**Issue by MarcoZehe** [podlove/podlove-publisher#717]
Right now, the buttons in the web player are links or button elements with some Unicode characters or CSS generated background images that represent the function. The title attribute is then used to pop over a bubble telling users what that control actually does. This setup is not ideal for blind screen reader users wanting to use the web player for listening. The title attribute is, because of so many cases of abuse, not a primary source of information for assistive technologies.

To solve this the easiest way, an aria-label attribute should be added to the same element that has the title attribute, and the contents should be the same string as for the title. That way, screen readers will immediately pick this up.

Alternatively, if using ARIA is not desired, a child span element should be added containing the text, and then moved out of the view port via CSS to make it invisible to people who can see. An example of which is the current hot method of making text available to screen readers, but not to sighted people, can be found [here](http://unobfuscated.blogspot.de/2015/01/updated-offscreen-techniques.html).

**Comment by eteubert** [comment]
Not sure what the status of this is in the new web player project. @line-o ? :)

**Comment by eteubert** [comment]
I think this is actively being discusses within the player project now, so I'm closing here.
